src/controllers/album.js

```diff
@@ -0,0 +1,92 @@
+const db = require("../db/index");
+
+const createAlbum = async (req, res) => {
+  const { id: artistId } = req.params;
```

Comment:
following on from my last comment, if you were to change the route like I suggested all you would need to change in this function is this line to:
```
const { artistId } = req.params;
```